### ISSUE TITLE
Mame 0.226: Translations: Minor Portuguese_Brazil Message Issues

### ISSUE THREAD

**Issue by mascguy** [mamedev/mame#7510]
While building Mame translations, failures occur with Portuguese_Brazil:

```
language/Portuguese_Brazil/strings.po:1693:19: invalid control sequence
language/Portuguese_Brazil/strings.po:3024: end-of-line within string
language/Portuguese_Brazil/strings.po:3024: keyword "come" unknown
language/Portuguese_Brazil/strings.po:3024:5: syntax error
language/Portuguese_Brazil/strings.po:3024: keyword "a" unknown
language/Portuguese_Brazil/strings.po:3024: keyword "com" unknown
language/Portuguese_Brazil/strings.po:3024: keyword "algo" unknown
language/Portuguese_Brazil/strings.po:3024: keyword "mas" unknown
language/Portuguese_Brazil/strings.po:3024: keyword "o" unknown
language/Portuguese_Brazil/strings.po:3024: keyword "perde" unknown
language/Portuguese_Brazil/strings.po:3025: end-of-line within string
language/Portuguese_Brazil/strings.po:3031: end-of-line within string
language/Portuguese_Brazil/strings.po:3031: keyword "sem" unknown
language/Portuguese_Brazil/strings.po:3031:4: syntax error
language/Portuguese_Brazil/strings.po:3031: keyword "um" unknown
language/Portuguese_Brazil/strings.po:3031: keyword "item" unknown
language/Portuguese_Brazil/strings.po:3031: keyword "e" unknown
language/Portuguese_Brazil/strings.po:3031: keyword "depois" unknown
language/Portuguese_Brazil/strings.po:3031: keyword "voc" unknown
language/Portuguese_Brazil/strings.po:3031: keyword "o" unknown
language/Portuguese_Brazil/strings.po:3031: keyword "consegue" unknown
language/Portuguese_Brazil/strings.po:3031: keyword "por" unknown
language/Portuguese_Brazil/strings.po:3031: keyword "exemplo" unknown
language/Portuguese_Brazil/strings.po:3032: end-of-line within string
msgmerge: found 24 fatal errors
```

The lines in question are:

```
[...]
msgstr "Cab. Mag.\Imperfeito\n"
[...]
msgstr "Use isto se quiser fazer um poke o Slot 1 (Quando você
começa com algo mas o perde)"
[...]
msgstr "Use isto se quiser fazer um poke com o valor do Último Slot (Quando você começa
sem um item e depois você o consegue por exemplo)"
[...]
```

The first message appears to be missing a 't', immediately following the first backslash. While the latter two messages appear to be split across lines, without an appropriate leading and trailing double quote per-line.

Changing those as follows, fixes the issue:

```
[...]
msgstr "Cab. Mag.\tImperfeito\n"
[...]
msgstr "Use isto se quiser fazer um poke o Slot 1 (Quando você "
"começa com algo mas o perde)"
[...]
msgstr "Use isto se quiser fazer um poke com o valor do Último Slot (Quando você começa "
"sem um item e depois você o consegue por exemplo)"
[...]
```

**Comment by mascguy** [comment]
Created new pull request with fixes: #7511

**Comment by mascguy** [comment]
Pull request merged, closing issue.
